contributing/code/bc.rst

```diff
@@ -204,38 +204,38 @@ Remove private property                             Yes
 Add constructor without mandatory arguments         Yes [1]_
 Remove constructor                                  No
 Reduce visibility of a public constructor           No
-Reduce visibility of a protected constructor        No
+Reduce visibility of a protected constructor        No [7]_
 Move to parent class                                Yes
 **Public Methods**
 Add public method                                   Yes
 Remove public method                                No
 Change name                                         No
 Reduce visibility                                   No
 Move to parent class                                Yes
-Add argument without a default value                No
-Add argument with a default value                   No
+Add argument without a default value                No [7]_
+Add argument with a default value                   No [7]_
 Remove argument                                     Yes [3]_
-Add default value to an argument                    No
-Remove default value of an argument                 No
-Add type hint to an argument                        No
-Remove type hint of an argument                     No
-Change argument type                                No
-Change return type                                  No
+Add default value to an argument                    No [7]_
+Remove default value of an argument                 No [7]_
+Add type hint to an argument                        No [7]_
+Remove type hint of an argument                     No [7]_
+Change argument type                                No [7]_
+Change return type                                  No [7]_
 **Protected Methods**
 Add protected method                                Yes
-Remove protected method                             No
-Change name                                         No
-Reduce visibility                                   No
+Remove protected method                             No [7]_
+Change name                                         No [7]_
+Reduce visibility                                   No [7]_
 Move to parent class                                Yes
-Add argument without a default value                No
-Add argument with a default value                   No
+Add argument without a default value                No [7]_
```

Comment:
I updated the notes to separate final classes and final methods.
A protected method in a final class is like a private method so this change is allowed, but it is not for a final method, so it should be ok now.